HelpSource/scdoc.js

```diff
@@ -190,11 +190,76 @@ function fixTOC() {
         });
     });
 
+    buildThemeSwitcher();
+
     if ($("#toc").length) {
         set_up_toc();
     }
 }
 
+// theming
+const scDocsThemeStorageKey = "scDocsTheme";
+const scriptLocation = document.currentScript.src;
+const themeNames = [
+    "classic",
+    "dark",
+    "default",
+    "dracula",
+    "solarizedDark",
+    "solarizedLight"
+];
+
+function buildThemeSwitcher() {
+    create_menubar_item("Theme \u25bc", "#", function (a, li) {
+        var indexes_menu = $("<div>", { class: "submenu" }).hide()
```

Comment:
minor thing, but could you rename this to themes_menu instead of indexes_menu?

Reply:
good catch - I decided to name it `themesMenu` which is not in style of `scdoc.js` but is more consistent with JS. I hope we can soon refactor the whole JS code nonetheless.